### ISSUE TITLE
Enhance expect-webdriverio with useful network assertions

### ISSUE THREAD

**Issue by christian-bromann** [webdriverio/webdriverio#5486]
Asserting Mocks with the help of `expect-webdriverio`

```js
const mock = browser.network.mock('/foo/**/bar')
expect(mock).toBeCalledTimes(1)
expect(mock).toBeCalledWithResponse({ foo: 'bar' })
// etc.
```

**Comment by christian-bromann** [comment]
See https://github.com/webdriverio/expect-webdriverio/pull/133

**Comment by christian-bromann** [comment]
This has landed and was released as `v1.3.0`.
